loleaflet/src/main.js

```diff
@@ -26,6 +26,8 @@ if (reuseCookies !== '') {
 
 var filePath = getParameterByName('file_path');
 var permission = getParameterByName('permission') || 'edit';
+// permission
+// The document's permission.
```

Comment:
This should be placed where permission **option** is declared in the first place

Reply:
@pedropintosilva It was added after line number **28**, this is the place where the **declaration** of permission was done